AttributeError in Registry class when asserting

I had a situation where I wanted to recreate the SQLAlchemyObjectType, creating a  new type. This does not work out of the box with Registry (which is OK), and an assert is thrown.

However, the assert itself has an problem:

```
..../api/graphql/types.py:74: in __init_subclass_with_meta__
    registry.register(cls)
../../../venv/lib/python3.6/site-packages/graphene_sqlalchemy/registry.py:12: in register
    'received "{}"'
E   AttributeError: 'tuple' object has no attribute 'format'
```

--- a/graphene_sqlalchemy/registry.py
+++ b/graphene_sqlalchemy/registry.py
@@ -8,7 +8,7 @@ class Registry(object):
     def register(self, cls):
         from .types import SQLAlchemyObjectType
         assert issubclass(cls, SQLAlchemyObjectType), (
-            'Only classes of type SQLAlchemyObjectType can be registered, ',
+            'Only classes of type SQLAlchemyObjectType can be registered, '
             'received "{}"'
         ).format(cls.__name__)
         assert cls._meta.registry == self, 'Registry for a Model have to match.'

--- a/graphene_sqlalchemy/tests/test_registry.py
+++ b/graphene_sqlalchemy/tests/test_registry.py
@@ -1,0 +1,31 @@
+import pytest
+
+from .models import Pet
+from ..registry import Registry
+from ..types import SQLAlchemyObjectType
+
+
+def test_register_incorrect_objecttype():
+    reg = Registry()
+
+    class Spam:
+        pass
+
+    with pytest.raises(AssertionError) as excinfo:
+        reg.register(Spam)
+
+    assert 'Only classes of type SQLAlchemyObjectType can be registered' in str(excinfo.value)
+
+
+def test_register_objecttype():
+    reg = Registry()
+
+    class PetType(SQLAlchemyObjectType):
+        class Meta:
+            model = Pet
+            registry = reg
+
+    try:
+        reg.register(PetType)
+    except AssertionError:
+        pytest.fail("expected no AssertionError")

